packages/alchemy/src/middleware/types/simulate-uo/interfaces.ts

```diff
@@ -0,0 +1,21 @@
+import type { Address } from "viem";
+import type { SimulateAssetType } from "./enums.js";
+
+export interface SimulateAssetChangesError extends Record<string, any> {
+  message: string;
+}
+
+export interface SimulateAssetChange {
+  assetType: SimulateAssetType;
+  changeType: string;
+  from: Address;
+  to: Address;
+  rawAmount?: string;
+  amount?: string;
```

Comment:
can we use `Hex` for these?

Reply:
i can check, but largely grabbed these from @mokok123 's definition.

Reply:
if the expectation is `Hex` agree with @denniswon let's put that here. It will be clearer to devs what the string value represents

Reply:
ah these aren't hex, they are literally string representations of decimals/numbers. going to stick with string

<img width="629" alt="image" src="https://github.com/alchemyplatform/aa-sdk/assets/43521356/312031d5-3f17-4fe3-bb61-8e764f09cb07">